src/test/java/io/kestra/plugin/serdes/avro/infer/FullIonAvroConversionTest.java

```diff
@@ -0,0 +1,136 @@
+package io.kestra.plugin.serdes.avro.infer;
+
+import com.amazon.ion.system.IonSystemBuilder;
+import com.google.common.collect.ImmutableMap;
+import io.kestra.core.junit.annotations.KestraTest;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.core.storages.StorageInterface;
+import io.kestra.core.utils.IdUtils;
+import io.kestra.core.utils.TestsUtils;
+import io.kestra.plugin.serdes.avro.AvroToIon;
+import io.kestra.plugin.serdes.avro.InferAvroSchemaFromIon;
+import io.kestra.plugin.serdes.avro.IonToAvro;
+import jakarta.inject.Inject;
+import org.apache.commons.collections.IteratorUtils;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@KestraTest
+public class FullIonAvroConversionTest {
```

Comment:
You should add a test to make sure that an Avro Schema is created for a canonical form of a JSON. i.e., : 
Inferred schema for `{"foo":"bar", "bar": "foo"}` must be the same than `{"bar":"foo", "foo": "bar"}` 

To check if two Avro schemas are identical, use `SchemaNormalization.parsingFingerprint64(schema());`

Reply:
test added